cogs5e/models/sheet/player.py

```diff
@@ -274,14 +312,15 @@ def full_str(self):
     def __str__(self):
         _max = self.get_max()
 
+        out = self.desc + '\n' if self.desc else ''
```

Comment:
This'll put the description (up to 800 characters) in *every* location the counter is stringified. That doesn't seem necessary - for example, if I had an alias 
```
!alias wildmagic embed <drac2>
out = [...]
# some wild magic code here
out.append(f'-f "Some Resource|{{character().cc_str('Some Resource')}}"')
return '\n'.join(out)
```

I wouldn't want the description to appear.